packages/portal/src/components/stories/StoriesTagsDropdown.vue

```diff
@@ -41,9 +44,11 @@
       >
         <RelatedCategoryTags
           v-if="displayTags.length > 0"
+          ref="relatedCategoryTags"
           :tags="displayTags"
           :selected="selectedTags"
           :heading="false"
+          tabindex="-1"
```

Comment:
I don't think this would have any effect as this is a div which receives no focus by default.

Reply:
It fixes it in dev mode, where for some reason the div does get focus.